test/e2e/config/ironic.yaml

```diff
@@ -28,11 +28,13 @@ variables:
   DEPLOY_IRONIC: "true"
   DEPLOY_BMO: "true"
   DEPLOY_CERT_MANAGER: "true"
+  IRSO_KUSTOMIZATION: "data/ironic-standalone-operator/operator"
```

Comment:
So do I understand correctly that  this will deploy `IRSO operator` and the other variable two lines down will bring in the ironic itself? Are we then handling the IRSO operator upgrade in these tests as well?

Reply:
Yep that makes sense, but I am thinking from the point of view of upgrading BMO, whether we should also upgrade IrSO alongwith, for now if I understand correctly IrSO version will remain static, in IrSO upgrade if/when we test upgrade does it also test BMO deployments ?